qiskit/opflow/gradients/derivative_base.py

```diff
@@ -232,3 +238,9 @@ def _factor_coeffs_out_of_composed_op(cls, operator: OperatorBase) -> OperatorBa
 
         else:
             return operator
+
+
+def _coeff_derivative(coeff, param):
```

Comment:
At first sight looking at the code  it seems like the body here could have become the body of the old method since they seem to be called in the same places in the code the same way. Maybe this covers more types with coeff, but since it has no typehints on params its hard to tell for me. I guess this is just intended to be used inside the gradients pkg here whereas the staticmethod had broader visibility.

Reply:
The `DerivativeBase.parameter_expression_grad` is old Aqua code that was moved to `ParameterExpression.gradient` some time ago but we never got around to removing the old part 🙂